### ISSUE TITLE
Remove duplicate IDs throughout site

### ISSUE THREAD

**Issue by NomeQ** [Trillium-Solutions/mendocino-wp-theme#6]
Every element ID should be unique on a given page. I found multiple of `home-news-arhive-link` on the homepage and `route-connections-box` on the route page. Remove the ID and use classes instead.
